README.md

```diff
@@ -70,38 +70,41 @@ In case you don't want to use the precompiled version, you can build Realm yours
     brew install android-ndk-r10e
     ```
 
- * Add two environment variables to your profile:
+ * Add two environment variables to your profile (presuming you used brew to install the NDK):
 
     ```
     export ANDROID_HOME=~/Library/Android/sdk
     export ANDROID_NDK_HOME=/usr/local/Cellar/android-ndk-r10e/r10e
     ```
 
- * If you want to build with Android Studio, `ndk.dir` has to be defined in the `realm/local.properties` as well.
+ * If you want to build with Android Studio, `ndk.dir` has to be defined in the `realm/local.properties` as well.  Note that there is a `local.properites` in the root directory that is *not* the one that needs to be edited.  Again, presuming you used brew to install the NDK:
 
     ```
     ndk.dir=/usr/local/Cellar/android-ndk-r10e/r10e
     ```
 
- * If you are using OS X, you'd be better to add following lines to `~/.profile` (or `~/.zprofile` if the login shell is `zsh`) in order for Android Studio to see those environment variables.
+ * If you are using OS X, you must also define, export and launchctl the following two symbols, so that Android Studio can see them.
```

Comment:
This is only true if you launch Android Studio from the launchpad, if you do it using the terminal it isn't needed. Also I believe the are issues with other shells like zsh, which @nhachicha uses.

That said, most people probably launch from the launchpad, this wording is probably fine.